### PR TITLE
Support classy-node in place of classy-api-client

### PR DESCRIPTION
**Pull request by brownjava**
Offers support to create and configure a classy-node client in place of classy-api-client.

### CODE REVIEW

File: src/DataSources/Clients.js [hunk 1]
```diff
@@ -20,13 +20,21 @@ class ClientDataSource {
...
 
     if (await config.get('api')) {
-      this.clients.apiClient = Promise.promisifyAll(require('classy-api-client')({
-        clientId: await config.get('APIV2_CLIENT_ID'),
-        clientSecret: await config.get('APIV2_CLIENT_SECRET'),
-        timeout: await config.get('api.timeout'),
-        oauthUrl: await config.get('api.oauthUrl'),
-        apiUrl: await config.get('api.apiUrl')
-      }));
+      if (await config.get('api.clientType') === 'classy-node') {
```
Comment (timothyk7) [inline src/DataSources/Clients.js]: *Still Researching

We should include the internal classy node to have access to all APIv2 methods

https://bitbucket.org/stayclassy/classy-node-internal-extension/src/master/

Comment (timothyk7) [reply src/DataSources/Clients.js]: all endpoints added
https://bitbucket.org/stayclassy/classy-node-internal-extension/src/master/src/resources.json

Comment (brownjava) [reply src/DataSources/Clients.js]: I'm not sure we can do that and still have this be an external npm-hosted project.  Maybe it's time to make pieces of this non-external?

**Comment by brownjava** [comment]
I'm abandoning this.  Classy node != classy api client.  The interfaces and the modes of using them are totally different.

We may want to add Classy node at some point, but I don't think we will really ever want these to something where you can switch between implementations by using a config file.

**Pull request closed without merging**